kubetest/main.go

```diff
@@ -169,6 +171,8 @@ func defineFlags() *options {
 	flag.BoolVar(&o.noAllowDup, "no-allow-dup", false, "if set --allow-dup will not be passed to push-build and --stage will error if the build already exists on the gcs path")
 	flag.BoolVar(&o.nodeTests, "node-tests", false, "If true, run node-e2e tests.")
 	flag.BoolVar(&o.perfTests, "perf-tests", false, "If true, run tests from perf-tests repo.")
```

Comment:
Change this argument from bool to string and make it take which tool to run from perf-tests.

Reply:
I've removed `perf-tests-tool` flag. Tool will be passed as a part of `perf-tests-args`.
For all perf-tests related tests, `run-e2e.sh` from perf-test repository will be called.